SU-SWS/stanford_media
# Hsd8 529 audio embeding

# READY FOR REVIEW

# Summary
- Added support for audio_embed_field
- changes bundle suggestion to use plugins that reduce the dependencies on new modules
- bundle suggestion logic moved to plugins
- Replace plugin from audio_embed_field to eliminate the need for a client_id from soundcloud
- New audio media type (as optional)
- cleanup and code simplifying

# Needed By (Date)
- asap (philit is in need of audio)

# Urgency
- medium

# Steps to Test

1. checkout https://github.com/SU-HSDO/suhumsci/pull/148
1. `composer install --prefer-source`
1. checkout this branch
1. `blt drupal:sync --site[sitename]`
1. create/edit a basic page
1. in the wysiwyg open the media browser, click the "embed external content" and paste a link to a soundcloud file
1. verify it works and displays the soundcloud widget iframe.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
